packages/style-engine/class-wp-style-engine.php

```diff
@@ -646,10 +570,10 @@ function wp_style_engine_get_styles( $block_styles, $options = array() ) {
 	}
 
 	if ( ! empty( $parsed_styles['declarations'] ) ) {
-		$styles_output['css']          = $style_engine::compile_css( $parsed_styles['declarations'], $options['selector'] );
+		$styles_output['css']          = WP_Style_Engine::compile_css( $parsed_styles['declarations'], $options['selector'] );
 		$styles_output['declarations'] = $parsed_styles['declarations'];
 		if ( true === $options['enqueue'] ) {
```

Comment:
`enqueue` doesn't really work as an option anymore since all the style engine is doing is "storing"

Maybe we can default the "context" to `null` and store wherever a non-empty context string value is passed (?)

Then we could remove `enqueue` altogether for now.

I'll do this in a follow up if folks think its legit.